tests/test_runner/test_discover_runner.py

```diff
@@ -294,6 +294,12 @@ def test_excluded_tags_displayed(self):
             runner.build_suite(['test_runner_apps.tagged.tests'])
             self.assertIn('Excluding test tag(s): bar, foo.\n', stdout.getvalue())
 
+    def test_no_of_tests_displayed(self):
+        runner = DiscoverRunner(verbosity=1)
+        with captured_stdout() as stdout:
+            runner.build_suite(test_labels=['foo', 'bar'])
```

Comment:
I would do what the other tests do and pass `test_labels` as a positional argument.

Also, to be clearer what `foo` and `bar` are doing, it might be better to call them something like `notfound1` and `notfound1`. I'm assuming it's finding two failed test instances for labels not found. Alternatively, you could find real tests by passing something starting with `'test_runner_apps...'`.

Reply:
You didn't address these comments.

Reply:
I think it is ok to name the test_labels as 'foo' or 'bar' since they are for testing purposes only.

Reply:
I agree with Chris, please use existing labels, e.g. `['test_runner_apps.sample', 'test_runner_apps.simple']`

Reply:
@girishsontakke The other part of my comment was to pass `test_labels` positionally for consistency with the other tests, so `runner.build_suite([...])` instead of `runner.build_suite(test_labels=[...])`.